pygeoapi/api.py

```diff
@@ -569,9 +569,10 @@ def get_collection_items(self, headers, args, dataset, pathinfo=None):
             LOGGER.error(exception)
             return headers_, 500, json.dumps(exception)
 
-        LOGGER.debug('processing property parameters')
+        LOGGER.debug('processing property parameters if enabled')
```

Comment:
Can we revert this line?  Then +1 to merge

Reply:
Done